main.go

```diff
@@ -111,29 +115,40 @@ func cloneRepo(cfg repoConfig) error {
 		HTTPUsername: string(cfg.GitHTTPUsername),
 		HTTPPassword: string(cfg.GitHTTPPassword),
 	}); err != nil {
-		return err
+		return newStepError(
+			"activate_git_http_credentials_failed",
+			err,
+			fmt.Sprintf("Activating Git HTTP credentials for %s failed", cfg.RepositoryURL),
+		)
 	}
 
 	// Git clone
 	logger := logv2.NewLogger()
 	envRepo := env.NewRepository()
 
-	tracker := tracker.NewStepTracker(envRepo, logger)
```

Comment:
Goland was complaining because the variable name (`tracker`) was actually the package name as well. So I renamed it.